src/cli/install.rs

```diff
@@ -8,6 +8,7 @@ use std::path::PathBuf;
 pub struct Args {
     /// The path to 'pixi.toml'
     #[arg(long)]
+    #[clap(num_args = 1..)]
```

Comment:
What does `num_args` mean here? It can take 1 or more arguments?

Reply:
This was weird, removed that